db/db_impl.h

```diff
@@ -1379,6 +1379,7 @@ class DBImpl : public DB {
   bool GetIntPropertyInternal(ColumnFamilyData* cfd,
                               const DBPropertyInfo& property_info,
                               bool is_locked, uint64_t* value);
+  bool HandleImmutableOptionStatistics(std::string* value);
```

Comment:
This is a function of DBImpl. This function name is not clear enough under this class. Mention something related to property. Also, I suggest we don't mention "ImmutableOption" here. statistics is statistics. The reason it is in ImmutableOption, is that we decided to currently put it there. It can be moved as we wish in the future. If we do that, we have to remember to update this function name. Otherwise, it will be invalid.